include/enums.h

```diff
@@ -1236,6 +1235,7 @@ typedef s8 ActionState;
 #define ActionState_STEP_DOWN          0x00000009
 #define ActionState_LAND               0x0000000A
 #define ActionState_STEP_DOWN_LAND     0x0000000B
+// Following states prohibit normal movement:
```

Comment:
Similar to how I feel about comments in the middle of structs, I'd prefer this at the end of the previous line rather than on its own line